src-js/app.js

```diff
@@ -1,3 +1,77 @@
 import * as Cookie from 'js-cookie';
+import * as lodash from 'lodash.throttle';
 
 window.Cookie = Cookie;
+
+// Instant search initialization
```

Comment:
Could we put the instantsearch stuff into a separate file and import it in here? I think https://github.com/circleci/circleci-docs/pull/2744 shows a good path forward for doing that

Reply:
sure